src_c/alphablit.c

```diff
@@ -25,6 +25,20 @@
 #define NO_PYGAME_C_API
 #include "_surface.h"
 
+/* See if we are compiled 64 bit on GCC or MSVC */
+#if _WIN32 || _WIN64
+   #if _WIN64
+     #define ENV64BIT
+   #endif
+#endif
+
+// Check GCC
+#if __GNUC__
+  #if __x86_64__ || __ppc64__
```

Comment:
I guess there are other compilers/systems with 64bitness.
The existing code detects 64bit for clang as well (I guess it defines `__GNUC__`).

So I don't know if something like below is necessary.

```c
/* stdint.h is missing from some versions of MSVC. */
#ifdef _MSC_VER
   #if _WIN64
     #define ENV64BIT
   #endif
#elif __GNUC__ && (__x86_64__ || __ppc64__)
     #define ENV64BIT
#else

#include <stdint.h>

#if INTPTR_MAX == INT64_MAX
#define ENV64BIT
#elif INTPTR_MAX == INT32_MAX
#define ENV32BIT
#else
#error Unknown pointer size or missing size macros!
#endif
#endif /* _MSC_VER */
```